core/cmd/local_client.go

```diff
@@ -552,7 +552,9 @@ func (cli *Client) RebroadcastTransactions(c *clipkg.Context) (err error) {
 	if err != nil {
 		return cli.errorOut(err)
 	}
-	ec := txmgr.NewEthConfirmer(app.GetSqlxDB(), ethClient, chain.Config(), keyStore.Eth(), keyStates, nil, nil, chain.Logger())
+
+	orm := txmgr.NewORM(app.GetSqlxDB(), lggr, cli.Config)
+	ec := txmgr.NewEthConfirmer(orm, ethClient, chain.Config(), keyStore.Eth(), keyStates, nil, nil, chain.Logger())
```

Comment:
Just noting, this pattern of an outside code directly instantiating/owning/calling Confirmer, is bad.
Ideally, confirmer should only be used/accessed by the TxMgr.

Anyways, this whole cmdline itself may not be used, and old. So maybe we will phase this out later.

Nothing to do in this PR though

Reply:
What is local client anyway?

Reply:
Just an entry point from the main application for Chainlink Node.